ext/node/polyfills/os.ts

```diff
@@ -247,6 +249,11 @@ export function version(): string {
   return Deno.osRelease();
 }
 
+/** Returns the machine type as a string */
+export function machine(): string {
+  return Deno.build.arch;
```

Comment:
nit: the output of os.machine() is different by node:

```
➜  deno git:(node_os_machine) ✗ cat t.mjs && node --version && node t.mjs
───────┬────────────────────────────────────────────────────────────────────────
       │ File: t.mjs
───────┼────────────────────────────────────────────────────────────────────────
   1   │ import os from "node:os";
   2   │
   3   │ console.log(os.machine());
───────┴────────────────────────────────────────────────────────────────────────
v21.5.0
arm64
➜  deno git:(node_os_machine) ✗ ./target/debug/deno run t.mjs
aarch64
```

Reply:
Fixed so it matches the machine reported by `uname -a`